src/Spaces/spectralelement.jl

```diff
@@ -128,6 +160,7 @@ struct SpectralElementSpace2D{
     ghost_dss_weights::D
     internal_surface_geometry::IS
     boundary_surface_geometries::BS
+    perimeter::P
```

Comment:
you don't really need to store this, as it's a singleton object (i.e. it doesn't take any space). But no harm in it.

Reply:
if you do add it, you then need to add it to the default constructors (that's the cause of the failing CI)

but a better option might be to just define:
```julia
perimeter(space::SpectralElementSpace2D) = 
Perimeter2D(Quadratures.degrees_of_freedom(space.quadrature_style))
```